wiki/Rules/el.md

```diff
@@ -0,0 +1,79 @@
+4.  Η υπερβολική χυδαιότητα δεν είναι ευπρόσδεκτη. Οι βρισιές ή οι βλασφημίες επιτρέπεται να χρησιμοποιούνται, αλλά όχι αν αρχίζουν να παραβιάζουν τους κανόνες \#2 και \#3.
+5.  Η κοινότητά μας είναι για όλες τις ηλικίες. Αυτό σημαίνει ότι περιεχόμενο 18+/NSFW όπως η χρήση ναρκωτικών ή απροκάλυπτα σεξουαλικά θέματα δεν είναι ευπρόσδεκτα εδώ. Εξαιρέσεις γίνονται ανάλογα με την περίπτωση, αλλά σε γενικές γραμμές, κρατήστε το καθαρό.
+
+Forum Κανόνες
+-------------
+
+1.  Μην κάνετε spam και μη γεμίζετε τα forum με άχρηστες αναρτήσεις/περιεχόμενα. Εάν δεν συμβάλλετε ουσιαστικά σε μια συζήτηση, καλύτερα να μη σχολιάσετε καθόλου.
+2.  Μπορείτε να στείλετε link με NSFW (ακατάλληλο) υλικό ΜΟΝΟ εάν είναι σκόπιμο σε μια συζήτηση, και μόνο αν το έχετε επισημάνει ως NSFW, και δεν είναι ενσωματωμένο σε ένα post.
+3.  Απαγορεύονται οι προσωπικές επιθέσεις. Οι κριτικές όλων των ειδών είναι ευπρόσδεκτες και ενθαρρύνονται, αλλά οι επιθέσεις προσωπικά σε άλλους χρήστες, όχι. Εάν δεν μπορείτε να αποδήξετε το επιχείρημά σας χωρίς τέτοιες τακτικές, καλύτερα να μην σχολιάσετε καθόλου.
+4.  Απαγορεύεται να κάνετε post σε πολύ παλιά και αδρανή threads, χωρίς περιεχόμενο στο post σας. Τέτοια πράξη μπορεί να οδηγήσει σε αυστηρή τιμωρία, αν γίνει επανειλημμένα.
+5.  Η αποστολή κακόβουλων συνδέσμων malware/adware/spyware/λογισμικών με ψευδή πρόθεση ή περιγραφή απαγορεύεται. Σε περίπτωση αμφιβολίας, ρωτήστε έναν συντονιστή (moderator).
+6.  Τα signatures δεν θα πρέπει να υπερβαίνουν τα 265kb στο συνολικό μέγεθος αρχείου. Οι κινούμενες εικόνες επιτρέπονται, εφ 'όσον είναι λογικές και δεν περιέχουν φωτοευαίσθητα στοιχεία (γρήγορη αναλαμπή κ.λπ.).
+7.  Τα κινούμενα avatar επιτρέπονται, αλλά υπο τους ίδιους όρους που ισχύουν και τα signatures. Τα avatar πρέπει πάντα να είναι κατάλληλα για όλες τις ηλικίες.
+8.  Οι διαφημίσεις και οι σύνδεσμοι (links) με σκοπό την διαφήμιση απαγορεύονται ρητά.
+9.  Περιεχόμενα που φορτώνουν αυτόματα (youtube βίντεο, εφαρμογές flash κ.λπ.) δεν επιτρέπονται.
+
+BSS Forum Κανόνες
+-----------------------------------
+
+Aυτοί οι κανόνες ισχύουν μόνο για τα [BSS](/wiki/Glossary) ([Beatmap](/wiki/Glossary) Submission System) threads. Αυτά τα threads δημιουργούνται αυτόματα μέσω του osu! client όταν ένα map υποβάλλεται μέσω του [editor](/wiki/Beatmap_Editor).
+
+1.  Μην δημιουργείτε threads στα [Pending Submissions](/wiki/Beatmaps) και [WIP](/wiki/Glossary) subforum. Χρησιμοποιήστε το κατάλληλο subforum όταν δημιουργείτε ένα thread. (Αυτό προφανώς αποκλείει τα BSS threads.)
+2.  Μην βγαίνεται εκτός θέματος στα BSS threads. Κάθε BSS thread αφορά ένα συγκεκριμένο beatmap. Αν έχετε κάποιο πρόβλημα που δεν σχετίζεται άμεσα με αυτό, κάντε το post στο κατάλληλο forum.
+3.  Μην κλέβετε beatmap αλλωνών, παρουσιάζοντάς τα σαν να είναι δικά σας. Επίσης, μην ανεβάζετε και μην χρησιμοποιήσετε έργα αλλωνών χωρίς τη ρητή άδειά τους (συμπεριλαμβανομένoυ, αλλά όχι περιορισμένα σε skins και guest difficulties).
+
+Off-Topic Κανόνες
```

Comment:
I prefer to leave it as it is without translation. Mainly because it's a sub-forum name and translating it would create misunderstandings as to where these rules are applied to.